cookiecutter.json

```diff
@@ -1,13 +1,23 @@
 {
-  "full_name": "Daniel Roy Greenfeld",
+  "__gh_slug": "{{ cookiecutter.github_username }}/{{ cookiecutter.slug }}",
```

Comment:
This is a bug which will cause Cookiecutter to behave erratically or throw an error. This is because the `__gh_slug` element requires earlier items.